src/sparse_blas/backends/mklcpu/mklcpu_basic.cpp

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+* Copyright 2023 Codeplay Software Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions
+* and limitations under the License.
+*
+*
+* SPDX-License-Identifier: Apache-2.0
+*******************************************************************************/
+
+// MKLCPU backend includes
+// These includes define their own oneapi::mkl::sparse namespace with some of the types that are used here: matrix_handle_t, index_base.
+#include <oneapi/mkl/spblas.hpp>
+
+#include "oneapi/mkl/sparse_blas/detail/helper_types.hpp"
+#include "oneapi/mkl/sparse_blas/detail/mklcpu/onemkl_sparse_blas_mklcpu.hpp"
+
+// Include are set up so that oneapi::mkl::sparse namespace refers to the MKLCPU namespace in this file.
+// oneapi::mkl::sparse::detail namespace refers to the oneMKL interface namespace.
```

Comment:
I honestly did not understand what you mean by this comment here. Are you trying to clarify the difference between the "oneapi::mkl::sparse" namespaces in this oneMKL Interfaces repository as opposed to the Intel oneMKL product, and which one maps to which namespace in this file? Maybe try rephrasing this?